plugins/field-slider/package.json

```diff
@@ -41,6 +41,8 @@
     "@blockly/dev-scripts": "^0.20200403.6",
     "@blockly/dev-tools": "^0.20200313.7",
     "blockly": "^3.20200123.1",
+    "chai": "^4.2.0",
+    "mocha": "^7.1.2",
```

Comment:
There should be no need to add mocha here, as we're using blockly-scripts tests to run the mocha tests. Are you seeing something different?

Reply:
I think it should work without mocha here because blockly-scripts tests pulls in that dependency for us.